style.css

```diff
@@ -0,0 +1,347 @@
+/*SITE_WIDE*/
+
+body {
+    font: 300 14px/1.5 Montserrat, Helvetica, sans-serif;
+    letter-spacing: 0.4px;
+    margin: 0;
+}
+
+h1 {
+    font: 700 32px/1.2 Roboto, sans-serif;
+    letter-spacing: 1.5px;

```

Comment:
For properties that have the same value multiple times, look into CSS variables to improve maintainability of the code.